src/coreclr/src/jit/compiler.h

```diff
@@ -1630,7 +1630,9 @@ struct fgArgTabEntry
         return 0;
     }
 
-    // Get the number of bytes that this argument is occupying on the stack.
+    // Get the number of bytes that this argument is occupying on the stack,
+    // including padding up to the target pointer size for platforms
+    // where a stack argument can't take less.
```

Comment:
it could be confused when we have arguments that require more than a pointer size, for example:
call(arguments to occupy registers, byte a, struct with 16 bytes alignment b)
in this example we will have such offsets on arm64 not-apple:
```
a - offset 0, GetStackByteSize() = 8,
b - offset 16, GetStackByteSize() = 16.
```
so the space between offset 1 and 8 is padding for a, the space between 8 and 16 is an alignment for b.